gui-tk/wiring.tcl

```diff
@@ -500,6 +500,11 @@ proc canvasContext {x y X Y} {
     .wiring.context add command -label "Copy selection" -command "minsky.copy"
     .wiring.context add command -label "Save selection as" -command saveSelection
     .wiring.context add command -label "Paste selection" -command pasteAt
+	if {![catch {clipboard get -type UTF8_STRING}]} {
+		.wiring.context entryconfigure end -state normal
```

Comment:
This statement is redundant (by default, the previous line creates the menu item in normal state).

Reply:
Thanks, I'm leaving it simply as:

    .wiring.context add command -label "Paste selection" -command pasteAt
	if {[catch {clipboard get -type UTF8_STRING}]} {
		.wiring.context entryconfigure end -state disabled
	}